<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YCZFilterServiceDialogClass</class>
 <widget class="QDialog" name="YCZFilterServiceDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YCZFilterServiceDialog</string>
  </property>
  <widget class="QComboBox" name="cmbObPath">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>30</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnComfirm">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>已知数据：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowseObserved">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbUnobPath">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>70</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>未知数据：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowseUnob">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>已知数据品位字段：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbObVal">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>已知数据标签字段：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbObLbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>未知数据标签字段：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbUnobLbl">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输出路径：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowseOutput">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineOutputPath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
